apiary.apib

```diff
@@ -984,18 +994,44 @@ Retrieve all Business Asset Attribute Types for a specified Business Asset Type
 
 # Group Order
 
-## Order Create [/accounts/{accountId}/orders{?includeAudit}]
+## Order Collection [/accounts/{accountId}/orders{?includeAudit,supplierName,orderNumber}]
+
++ Parameters
+    + accountId: 1 (required, number) - {INT} ID of the account in the form of an integer
```

Comment:
Most other lines in the file use a different order. They use type, then required / optional. Should these be switched?